frontend/src/pages/contacts/ContactStatusSelect.tsx

```diff
@@ -0,0 +1,72 @@
+import axios from 'axios';
+import React, { useState } from 'react';
+import { SelectFormControl } from '../../components/molecules/formControls/selectFormControl';
+import { uri } from '../../utils';
+
+interface Props {
+	scoreDigit: any;
+}
+
+const ContactStatusSelect: React.FC<Props> = ({ scoreDigit }) => {
+	const [selectedStatus, setSelectedStatus] = useState<string>('All');
+
+	const handleStatusChange = async (event: any) => {
+		const status = event.target.value;
+		setSelectedStatus(status);
+
+		let url = uri('contacts');
+		if (status !== 'All') {
+			url = uri(`contacts?status=${status}`);
+		}
+
+		try {
+			const response = await axios.get(url, {
+				headers: {
+					Authorization: `Bearer ${localStorage.getItem('access_token')}`,
+				},
+			});
+			scoreDigit(response.data);
+			console.log(response.data.contacts);
+		} catch (error) {
+			console.error(error);
+		}
```

Comment:
We don't use console logs for production. You are allowed to use them while you are developing. But remove them before pushing your code.

Reply:
Thanks, I'll fix it.